docs/source/usage/plugins/radiation.rst

```diff
@@ -306,7 +306,176 @@ Command line flag                        Output description
                                          These are for restart purposes and for more complex data analysis.
 ======================================== ========================================================================================================================
 
-Analysing tools
+
+Text-based output
+"""""""""""""""""
+
+The text-based output of ``lastRadiation`` and ``totalRadiation`` contains the intensity values in SI-units :math:`[Js]`. Intensity-values for different frequencies are separated by spaces, while newlines separate values for different observation directions. 
```

Comment:
Why dash in `Intensity-values`? In case it is supposed to be like that, maybe the earlier occurrence on the same line should be changed?

Reply:
@sbastrakov You are right. That should not be dashed in. Great catch.